src/charm.py

```diff
@@ -123,9 +123,11 @@ def _additional_hostnames(self) -> List[str]:
             The additional hostnames set by configuration already split by comma.
         """
         return [
-            x
-            for x in self._get_config_or_relation_data("additional-hostnames", "").split(",")
-            if x
+            hostname
+            for hostname in self._get_config_or_relation_data("additional-hostnames", "").split(
+                ","
+            )
```

Comment:
The indentation seems strange, wouldn't it be better to return to the line before ".split" ?

Reply:
Sure, changed. :)